infrastructure/output.tf

```diff
@@ -16,6 +16,10 @@ output "ENVELOPES_QUEUE_WRITE_CONN_STRING" {
   value = "${data.azurerm_key_vault_secret.envelopes_queue_send_conn_str.value}"
 }
 
+output "ENVELOPES_QUEUE_READ_CONN_STRING" {
+  value = "${data.azurerm_key_vault_secret.envelopes_queue_listen_conn_str.value}"
```

Comment:
is the secret config missing here or was already added ?

Reply:
It was already added in the recent pr https://github.com/hmcts/bulk-scan-orchestrator/pull/352